README.md

```diff
@@ -1,41 +1,46 @@
-# Website
+# Portals Knowledgebase
 
-This website is built using [Docusaurus](https://docusaurus.io/), a modern static website generator.
+## Development
 
-### Installation
+### Adding content
 
-```
-$ yarn
-```
+We use [Docusaurus](https://docusaurus.io/) within these docs.
 
-### Local Development
+- You can find a [general set of guides to Docusaurus here](https://docusaurus.io/docs/category/guides).
+- **Docusaurus provides extra cool formatting features** on top of vanilla Markdown,
+  see the [guide to these features here](https://docusaurus.io/docs/markdown-features).
 
-```
-$ yarn start
+### Running locally
+
+Install deps and run the dev server locally by running:
+
+```sh
+yarn
+yarn start
 ```
 
-This command starts a local development server and opens up a browser window. Most changes are reflected live without having to restart the server.
+Most changes are reflected live without having to restart the server.
 
-### Build
+## Building
 
-```
-$ yarn build
+```sh
+yarn build
 ```
 
 This command generates static content into the `build` directory and can be served using any static contents hosting service.
 
-### Deployment
+## Deploying
 
 Using SSH:
 
-```
-$ USE_SSH=true yarn deploy
+```sh
+USE_SSH=true yarn deploy
 ```
 
 Not using SSH:
 
-```
-$ GIT_USER=<Your GitHub username> yarn deploy
+```sh
+GIT_USER=<Your GitHub username> yarn deploy
 ```
```

Comment:
Deploying from the command line shouldn't be the default, we should recommend using the GitHub action to deploy once a PR is submitted/merged.